pkg/master/master.go

```diff
@@ -541,8 +550,14 @@ func (m *Master) init(c *Config) {
 		} else {
 			glog.Errorf("Failed to cast %v to HTTPKubeletClient, skipping SSH tunnel.")
 		}
+		healthzChecks = append(healthzChecks, healthz.NamedCheck("SSH Tunnel Check", m.IsTunnelSyncHealthy))
 	}
 
+	m.lastSyncMetric = prometheus.NewGaugeFunc(prometheus.GaugeOpts{
```

Comment:
Should this be inside the above if block? I don't know prometheus well enough to know the implications, but this metric only makes sense in the ssh-tunnel case.